x/perpetual/keeper/pool.go

```diff
@@ -157,7 +157,10 @@ func (k Keeper) GetBorrowInterestRate(ctx sdk.Context, startBlock, startTime uin
 				Quo(sdk.NewDec(numberOfBlocks)).
 				Quo(blocksPerYear)
 
-			return sdk.MaxDec(finalInterestRate.Mul(takeProfitBorrowFactor), k.GetParams(ctx).BorrowInterestRateMin)
+			return sdk.MaxDec(
```

Comment:
There is one if block above also we will have to add there as well

Reply:
updated